cmd/register_push.go

```diff
@@ -31,15 +34,15 @@ func newRegisterPushCmd(out io.Writer) *cobra.Command {
 			if err != nil {
 				return errors.Wrap(err, "initialize pubsub client")
 			}
-			return registerPush(cmd.Context(), out, pubsubClient, topicID, endpoint)
+			return registerPush(cmd.Context(), out, pubsubClient, topicID, endpoint, ackDeadlineNum)
 		},
 	}
 	command.SetOut(out)
 	return command
 }
 
 // registerPush registers new push endpoint
-func registerPush(ctx context.Context, out io.Writer, pubsubClient *pkg.PubSubClient, topicID, endpoint string) error {
+func registerPush(ctx context.Context, out io.Writer, pubsubClient *pkg.PubSubClient, topicID, endpoint string, ackDeadline int64) error {
```

Comment:
Now the `ackDeadline` param is expecting the number of seconds implicitly, so instead, passing `time.Duration` here would be more robust!